src/pymor/operators/basic.py

```diff
@@ -127,25 +127,25 @@ def apply_inverse(self, V, mu=None, least_squares=False):
                     raise InversionError(e)
             return R
 
-    def apply_inverse_transpose(self, U, mu=None, least_squares=False):
+    def apply_inverse_adjoint(self, U, mu=None, least_squares=False):
         from pymor.operators.constructions import FixedParameterOperator
         if not self.linear:
             raise LinAlgError('Operator not linear.')
         assembled_op = self.assemble(mu)
         if assembled_op != self and not isinstance(assembled_op, FixedParameterOperator):
-            return assembled_op.apply_inverse_transpose(U, least_squares=least_squares)
+            return assembled_op.apply_inverse_adjoint(U, least_squares=least_squares)
         else:
             # use generic solver for the transpose operator
             from pymor.operators.constructions import AdjointOperator
-            options = {'inverse': self.solver_options.get('inverse_transpose') if self.solver_options else None}
-            transpose_op = AdjointOperator(self, with_apply_inverse=False, solver_options=options)
-            return transpose_op.apply_inverse(U, mu=mu, least_squares=least_squares)
+            options = {'inverse': self.solver_options.get('inverse_adjoint') if self.solver_options else None}
+            adjoint_op = AdjointOperator(self, with_apply_inverse=False, solver_options=options)
+            return adjoint_op.apply_inverse(U, mu=mu, least_squares=least_squares)
 
     def as_range_array(self, mu=None):
         return self.apply(self.source.from_numpy(np.eye(self.source.dim)), mu=mu)
 
     def as_source_array(self, mu=None):
-        return self.apply_transpose(self.range.from_numpy(np.eye(self.range.dim)), mu=mu)
+        return self.apply_adjoint(self.range.from_numpy(np.eye(self.range.dim)), mu=mu)
```

Comment:
This is inconsistent with `NumpyMatrixOperator.as_source_array`. One of these should have an additional `.conj()`.

Reply:
Good point. I think that `as_source_array` should behave as implemented here. I have corrected `NumpyMatrixOperator.as_source_array` accordingly.

Reply:
I also prefer `OperatorBase.as_source_array` how it's implemented here.
With this resolved, should there be `.T` in `make_array` in [`OperatorInterface.as_source_array` docstring](https://github.com/pymor/pymor/blob/b268a2712d43c48bd2c928c11fbb70d2725b2cf6/src/pymor/operators/interfaces.py#L274)?

Reply:
Yeah, thats right ...